_releases/24.0.1.md

```diff
@@ -0,0 +1,419 @@
+---
+title: Bitcoin Core 24.0.1
+id: en-release-24.0.1
+name: release-24.0.1
+permalink: /en/releases/24.0.1/
+excerpt: Bitcoin Core version 24.0.1 is now available
+date: 2022-12-12
+
+## Use a YAML array for the version number to allow other parts of the
+## site to correctly sort in "natural sort of version numbers".
+## Use the same number of elements as decimal places, e.g. "0.1.2 => [0,
+## 1, 2]" versus "1.2 => [1, 2]"
+release: [24, 0, 1]
+
+## Optional magnet link.  To get it, open the torrent in a good BitTorrent client
+## and View Details, or install the transmission-cli Debian/Ubuntu package
+## and run: transmission-show -m <torrent file>
+#
+## Link should be enclosed in quotes and start with: "magnet:?
+optional_magnetlink: "magnet:?xt=urn:btih:d7604a67c8ed6e3b35da15138f8ac81d7618788c&dn=bitcoin-core-24.0.1&tr=udp%3A%2F%2Ftracker.openbittorrent.com%3A80&tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337%2Fannounce&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.leechers-paradise.org%3A6969%2Fannounce&tr=udp%3A%2F%2Fexplodie.org%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.torrent.eu.org%3A451%2Fannounce&tr=udp%3A%2F%2Ftracker.bitcoin.sprovoost.nl%3A6969"
```

Comment:
I couldn't verify the torrent files since it seems no one is currently seeding it.

Going forward, should we also add a [BTMH](https://en.wikipedia.org/wiki/Magnet_URI_scheme) checksum to avoid relying on SHA-1 hashes?

Reply:
I just whitelisted the hash on my tracker, but can't test it atm.

Reply:
Thanks @Sjors, I've now been able to download the torrent and verified it's all good.